### ISSUE TITLE
[question] What is causing my dataset to not reconstruct properly?

### ISSUE THREAD

**Issue by StellaGracia** [alicevision/meshroom#1308]
Hello! I am trying to reconstruct a small luggage. I am using a photo studio (static camera and luggage turned with a turntable). I have tried taking multiple sets of pictures but somehow half my images are always not reconstructed. 

This is the setup
![image](https://user-images.githubusercontent.com/44325657/109468013-fba53480-7aa6-11eb-9f60-a8aa04913883.png)

And how half of my images are not reconstructed, causing hollow top and bottom
![image](https://user-images.githubusercontent.com/44325657/109467444-2773ea80-7aa6-11eb-915b-82d905783591.png)

Here is the link to all the files related to this reconstruction https://drive.google.com/drive/folders/1R2STxavWxFPyHp_m4tXkVCvDcFAdaFuV?usp=sharing

I have tried cropping out non-moving features (like the corners of the photo studio) as well as adding a pattern paper below the luggage that turns with the luggage but unfortunately they dont work too.

What am I doing wrong? Appreciate any help I can get :)

**Comment by natowi** [comment]
> I have tried cropping out non-moving features (like the corners of the photo studio) as well as adding a pattern paper below the luggage that turns with the luggage but unfortunately they dont work too.

Yes, that is important. This could be voided by capturing close up images / choosing portrait mode.
Additionally you should experiment with the lighting as your luggage is highly reflective. Avoid reflections on the surface / capture one rotation of the lower part, then move the lighting and capture the other part so you don´t have reflecting surfaces in your images. Use indirect lighting. Also the luggage should fill most of your image not only 1/3

**Comment by StellaGracia** [comment]
> Btw what phone model do you use? Looks like it has not been recognized by Meshroom.

Thank you for your suggestions! I'll try them out and let you know :)

I am using Samsung Note 3. The model is actually in the Meshroom list. However, I am currently taking a video and sampling images from the videos. Is there a way I can still select the phone model manually in meshroom?

**Comment by natowi** [comment]
- You could try the keyframeselection node to extract images from mp4 videos with metadata.
add/or
- You could use the cameracalibration node to calibrate your camera from a video and copy the camera intrinsics from the cal file in the camerainit node
- there are also tools that allow you to batch edit image metadata. you could probably transfer the metadata from an image taken from the same device with identical settings

**Comment by StellaGracia** [comment]
> * there are also tools that allow you to batch edit image metadata. you could probably transfer the metadata from an image taken from the same device with identical settings

Hi again! Thank you so much for your inputs. I tried transferring image metadata to my images. However, when I add my new set of images, images with the same metadata are registered as one image (the first one only). I also tried manually capture image by image using burst shots and it also appears that images that are taken almost together are registered as one image. What am I doing wrong?

**Comment by natowi** [comment]
> that images that are taken almost together are registered as one image.

There should be a noticeable difference between the images. It is called StructureFromMotion: camera motion (or in your case turntable motion) is required to calculate the point cloud. Nearly identical camera positions don´t help with that.

In addition to that it sounds like there is still something wrong with your metadata. You could share a few images of your new datasets and I can check them for errors. 

For your dual camera setup, this may be interesting: https://github.com/alicevision/meshroom/wiki/Multi-Camera-Rig

**Comment by StellaGracia** [comment]
> But it sounds like there is still something wrong with your metadata. You could share a few images of your new datasets and I can check them for errors.

I dragged in these photos https://drive.google.com/drive/folders/1fo0gDVmvcxz7LurrHevW8QPB3O9nyeoB?usp=sharing 
They are actually taken as individual images.

However when I dragged into meshroom, 
![image](https://user-images.githubusercontent.com/44325657/110962287-c5f82980-838b-11eb-964f-328912565203.png)
It seems like only 2 images are registered.

What am I doing wrong?

**Comment by natowi** [comment]
I checked the metadata. I think the problem is that you cloned all the metadata, including the capturing time. Meshroom checks the metadata and assumes the images are duplicates, only importing one of them. This should probably be mentioned in the documentation ^.^

(There are cases, where we have identical metadata (including model and time): when we use a [synced camera rig](https://github.com/alicevision/meshroom/wiki/Multi-Camera-Rig). In that case it would be important to change the serialnumber for each device)
